### PR TITLE
Trivial: Spelling fix, python → Python

### PR DESCRIPTION
**Pull request by comradekingu**
As per https://www.python.org

### CODE REVIEW

**Comment by jonasschnelli** [comment]
See https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#refactoring
IMO it's a distracting PR with no benefits.

File: src/qt/bitcoinstrings.cpp [hunk 1]
```diff
@@ -219,7 +219,7 @@ QT_TRANSLATE_NOOP("bitcoin-core", ""
 "%s)"),
 QT_TRANSLATE_NOOP("bitcoin-core", ""
 "Username and hashed password for JSON-RPC connections. The field <userpw> "
-"comes in the format: <USERNAME>:<SALT>$<HASH>. A canonical python script is "
```
Comment (fanquake) [inline src/qt/bitcoinstrings.cpp]: Don't modify this file, it is "Automatically generated by extract_strings_qt.py".

**Comment by fanquake** [comment]
Please squash your commits, and amend the commit message as per the [contributing guidelines](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md).

**Comment by laanwj** [comment]
I appreciate that you're trying to contribute, and have nothing against this change in itself, however please don't submit PRs that just change words around. If you are looking for something worthwhile to address you could look at the open issues, for example the list of [good first issues](https://github.com/bitcoin/bitcoin/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) and pick something from there.

**Comment by comradekingu** [comment]
@laanwj However minuscule, this is a user facing string with a bug in it. Was trying to find out what the process is, for fixing some of the more unclear strings later.

**Comment by promag** [comment]
I wouldn't say bug, nothing is broken.

**Comment by laanwj** [comment]
Don't use the word 'bug' like that, please. It's certainly possible to have a bug in a text message or documentation when it explains things wrong, and sometimes those are serious bugs that can result in coin loss.
However, this is not one of such.
You're welcome to improve messages if they're unclear, though!

**Comment by comradekingu** [comment]
@laanqj My apologies. Adjusting for clarity in strings other than this leaves a whole lot less room for misunderstandings, making for fewer errors in translation.

**Comment by MarcoFalke** [comment]
@comradekingu No need to apologise.

Tend to NACK, as this will invalidate all existing translations and we'd have to manually (or by script) fix them up on transifex. Imo, not worth it.

**Comment by promag** [comment]
Please drop unicode from PR title and commit message.

**Comment by promag** [comment]
Good point @MarcoFalke.

**Pull request closed without merging**